composeApp/src/commonMain/kotlin/band/effective/office/elevator/ui/authorization/authorization_profile/AuthorizationProfileScreen.kt

```diff
@@ -95,6 +99,9 @@ fun AuthorizationProfileComponent(
     val borderColor2 = remember { mutableStateOf(textGrayColor) }
     val leadingColor2 = remember { mutableStateOf(textGrayColor) }
 
+    var personName by remember { mutableStateOf(state.name) }
```

Comment:
If we change name in state we can collect it change.  Why we need remember?

Reply:
fix due to a problem from here: fix due to a problem from [here](https://medium.com/androiddevelopers/effective-state-management-for-textfield-in-compose-d6e5b070fbe5)